app/utils/display-converter.js

```diff
@@ -34,7 +34,8 @@ const hideServiceNavTemplates = [
   'request-to-go-live/organisation-name',
   'request-psp-test-account/index',
   'two-factor-auth/index',
-  'two-factor-auth/configure'
+  'two-factor-auth/configure',
+  'kyc/organisation-url'
```

Comment:
I don't think we want to hide the service nav for this page? The add kyc details is done from within the context of a gateway account, so think it should be shown

Reply:
Good spot - I also added the side menu, as the mockup has this included.